integrationtesting/opendcs-tests/src/test/java/org/opendcs/fixtures/annotations/EnableIfTsDb.java

```diff
@@ -47,21 +47,37 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext ctx
                 {
                     Optional<AnnotatedElement> element = ctx.getElement();
                     EnableIfTsDb anno = element.get().getAnnotation(EnableIfTsDb.class);
-                    if (anno.impl().isEmpty())
+                    final String configName = configs.get(0).getName();
+                    if (anno.value().length == 0)
                     {
                         retVal = ConditionEvaluationResult.enabled("Is Timeseries Db");
                     }
-                    else if (anno.impl().equalsIgnoreCase(configs.get(0).getName()))
+                    else if (matches(anno.value(),configName))
```

Comment:
Are you OK that this matches() is case sensitive; different from previous version of this file.

Reply:
Hmm, didn't even notice that, good question. But I feel like those probably should be identical to the actual provider name.